nzic/past/2016/rules.md

```diff
@@ -0,0 +1,77 @@
+
+For power users, we suggest that the compiler be made available on the command line. This is already the case with Linux operating systems. For Windows, for which cmd.exe lacks many features and is difficult to use, MinGW (including a port of GCC) is recommended along with MSYS as a command-line interface. Cygwin is not recommended because it may be too slow.
+
+#### Documentation and other material during the contest
+
+While competing, students may only access the [https://train.nzoi.org.nz/](https://train.nzoi.org.nz/) website, or an official (static) language documentation site (language documentation installed locally may also be used). Electronic language documentation (but not source code) stored locally is also permitted. Forums or interactive websites are strictly forbidden.
+
+Students may also use any printed material during the contest, including language documentation, textbooks or source code. These materials may not be shared.
+
+Students may not access or use any electronically stored source code, whether it is stored locally or on the internet. This includes source code written by the student prior to the contest. All source code submitted must be written during the contest, without any collaboration.
+
+#### Help
+
+Students may ask the supervisor questions on administrative tasks, such as, when the contest ends or how to use the website to make a submission.
+
+Questions such as how to solve a problem, how use a language feature, or where a bug is in a program will not be answered. The student should be able to find out how to use a language feature by reading the language documentation.
+
+If a student is new to a programming language, we suggest that they bring printed sample source code for reference.
+
+Any questions related to the problems should be emailed to NZIC for a response.
+
+#### Leaving early
+
+This is up to each individual school or supervisor. However, the time at which any students leave the contest early will need to be reported.
+
+### Scoring
+
+There may be slight modifications to scoring. These will be finalized closer to the time of the contests.
+
+There will be official scoreboards for individual students, and for schools.
+
+Each student will be scored for the year based on a weighted sum of their round 1-4 scores. The higher scores will have a larger weighting than lower scores.
+
+Each school will receive a score based on the weighted average of their top 3 students in each round. The score for the year will be the sum of each round.
+
+Based on the year's scores, we will guarantee a place in our January camp for up to 10 of the highest scoring students (depends on number of students competing in the final round, and normal camp fee applies). Other students will still have a chance to apply to come to our camp. We will offer a place to as many interested students as we can.
```

Comment:
I'm not sure if this needs to be corrected, as it's not in the archive and we haven't got that paragraph in the new rules.